packages/trader/src/AppV2/Components/TradeParameters/Duration/__tests__/duration.spec.tsx

```diff
@@ -6,6 +6,8 @@ import { mockStore } from '@deriv/stores';
 import { TCoreStores } from '@deriv/stores/types';
 import userEvent from '@testing-library/user-event';
 import { useSnackbar } from '@deriv-com/quill-ui';
+import moment from 'moment';
```

Comment:
in wallets, we recently removed moment due to its size, 
as much as I like moment (and I personally still use it in non-client-facing projects, I'm wondering if it shouldn't be deprecated and not being added anymore)

Reply:
In V2 , we are not actually using moment . The `server_time` is handled in store primarily used in V1 and is handled via moment . So in order to utilize that I had to use the moment .